doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -2447,7 +2581,7 @@ IRIS-7398: The :ref:`nrf_cloud_multi_service` sample does not support using the
 
   **Affected platforms:** nRF9161
 
```

Comment:
.. rst-class:: v2-8-0

NCSDK-30050: The :ref:`nrf_cloud_rest_fota` sample experimental SMP FOTA support crashes with a secure fault approximately 1 out of 5 times.